lua/ulx/modules/sh/freeze.lua

```diff
@@ -0,0 +1,50 @@
+CATEGORY_NAME = "CFC"
+
+do
+    local function freezeProps( callingPlayer, targetPlayers ) 
+        local entits = ents.FindByClass( "prop_*" )
```

Comment:
entits? how about just naming `entities` or `ents` or `plyProps`

Reply:
at the time i was thinking there would be issues with that, but i dont think that anymore

Reply:
done